SCClassLibrary/Common/Collections/Array.sc

```diff
@@ -131,6 +131,14 @@ Array[slot] : ArrayedCollection {
 		_ArrayContainsSeqColl
 		^this.primitiveFailed
 	}
+	maxDepth { arg max = 1;
+		_ArrayMaxDepth
+		^super.maxDepth(max)
+	}
+	maxSizeAtDepth { arg rank;
+		_ArrayMaxSizeAtDepth
+		^super.maxSizeAtDepth(rank)
+	}
```

Comment:
It's a very basic readability and clarity issue. I'm not saying you have to add them everywhere else in the library.